plugins/org.orbisgis.h2gis/plugin.xml

```diff
@@ -29,23 +29,23 @@
                     description="H2GIS Embedded"
                     categories="spatial,sql,embedded">
 
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2/2.0.201-SNAPSHOT/h2-2.0.201-20211115.092925-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis/2.0.0-SNAPSHOT/h2gis-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-api/2.0.0-SNAPSHOT/h2gis-api-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-network/2.0.0-SNAPSHOT/h2gis-network-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-osgi/2.0.0-SNAPSHOT/h2gis-osgi-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-test-utilities/2.0.0-SNAPSHOT/h2gis-test-utilities-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2gis-utilities/2.0.0-SNAPSHOT/h2gis-utilities-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/postgis-jts/2.0.0-SNAPSHOT/postgis-jts-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/postgis-jts-osgi/2.0.0-SNAPSHOT/postgis-jts-osgi-2.0.0-20211115.133346-2.jar"/>
-                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/cts/1.6.0-SNAPSHOT/cts-1.6.0-20211110.101439-1.jar"/>
-                    <file type="jar" path="maven:/org.slf4j:slf4j-simple:1.7.20"/>
-                    <file type="jar" path="maven:/org.slf4j:slf4j-api:1.7.20"/>
+                    <file type="jar" path="https://oss.sonatype.org/content/repositories/snapshots/org/orbisgis/h2/2.1.209-SNAPSHOT/h2-2.1.209-20220104.144750-1.jar"/>
```

Comment:
use 206

Reply:
and the official one from maven
 <file type="jar" path="maven:/com.h2database:h2:2.0.206"/>
https://mvnrepository.com/artifact/com.h2database/h2/2.0.206